frontend/src/__tests__/cypress/cypress/e2e/modelServing/ServingRuntimeList.cy.ts

```diff
@@ -1,3 +1,4 @@
+import chaiSubset from 'chai-subset';
```

Comment:
Move this to `e2e.ts` so that every test gets it for free without any further setup.

```
import chaiSubset from 'chai-subset';
chai.use(chaiSubset);
```